wherehows-web/mirage/scenarios/default.ts

```diff
@@ -13,7 +13,7 @@ export default function(server: IMirageServer) {
   server.loadFixtures(...fixtures);
   server.create('config');
   server.createList('owner', 6);
-  server.createList('dataset', 10);
+  server.createList('dataset', 10, 'withHealth');
```

Comment:
why not make this an independent scenario? seems like unneeded coupling to the `dataset` response, other tests do not need this trait so it seems like it may make testing brittle if those tests change

Reply:
Ah. I think I see what you're going for in the other tests. I prefer the more brittle testing with coupling where there are entity relationships but will change to be consistent.

Reply:
I guess from an ember-data perspective, that would be workable. I just didn't want to get into a situation, where if the implementation for a feature changed that it's test needed updating, you'd have to go into a bunch of other tangential tests' setup and make updates there as well